static/js/tools/scatter/place_and_type_options.tsx

```diff
@@ -150,11 +155,12 @@ function PlaceAndTypeOptions(): JSX.Element {
               className="pac-target-input"
             >
               <option value="">Select a place type</option>
-              {childPlaceTypes.map((type) => (
-                <option value={type} key={type}>
-                  {type}
-                </option>
-              ))}
+              {childPlaceTypes &&
```

Comment:
was there a case you encountered where this list was empty? it seems like it should at least be set to [] by default.

Reply:
scatter app.test.tsx kept failing complaining about it being undefined .. even though I don't think I've actually ever seen it undefined